src/tests/nativeaot/SmokeTests/DwarfDump/DwarfDump.csproj

```
@@ -0,0 +1,14 @@
+
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestKind>BuildAndRun</CLRTestKind>
+    <CLRTestPriority>0</CLRTestPriority>
+    <!-- Test checks symbols, so don't run in non-debug builds. -->
+    <CLRTestTargetUnsupported Condition="'$(Configuration)' != 'Debug' or '$(TestBuildMode)' != 'nativeaot' or '$(TargetOS)' != 'linux'">true</CLRTestTargetUnsupported>
```

Nit: We don't run any of the `SmokeTests` outside NativeAOT. Few would be compatible with other runtimes - we often test very specific things.

```suggestion
    <CLRTestTargetUnsupported Condition="'$(Configuration)' != 'Debug' or '$(TargetOS)' != 'linux'">true</CLRTestTargetUnsupported>
```